cpp/order/Steinhardt.h

```diff
@@ -65,8 +65,8 @@ class Steinhardt
      *                         Allows looking at, for instance, only the second shell,
      *                         or some other arbitrary rdf region.
      */
-    Steinhardt(float rmax, unsigned int l, float rmin = 0, bool average = false, bool Wl = false)
-        : m_Np(0), m_rmax(rmax), m_l(l), m_rmin(rmin), m_average(average), m_Wl(Wl), m_Qlm_local(2 * l + 1)
+    Steinhardt(float rmax, unsigned int l, float rmin = 0, bool average = false, bool Wl = false, bool weighted = false)
```

Comment:
For 2.0 we should probably do a review to make sure that default parameter values are properly mirrored everywhere in Cython/C++. It's annoying, but probably the best approach if we want to move towards allowing C++ usage of freud.

Just a note, not a to-do.